core/src/init/button/button.ts

```diff
@@ -1,6 +1,7 @@
 import { border, Button, ProgressCircle } from "@moai/core";
 import outset from "./outset.module.css";
 import flat from "./flat.module.css";
+import s from "./size.module.css";
```

Comment:
"s" should only be used for the generic "style.module.css". For specific names like "size.module.css", we should be explicit:
```suggestion
import size from "./size.module.css";
```